src/utils/time-entry-utils/combineTimeErrors.spec.jsx

```diff
@@ -0,0 +1,86 @@
+import TimeInputErrors from '../../components/timecard/time-input-errors/TimeInputErrors';
+import { clashingProperties, inputNames } from '../constants';
+import { combineExistingAndTimeClashErrors } from './combineTimeErrors';
+
+describe('combineExistingAndTimeClashErrors', () => {
+  it('should return an empty array when no errors are passed in', async () => {
+    var result = combineExistingAndTimeClashErrors({}, {}, null, null);
```

Comment:
Quibble: Is there any reason to use `var` instead of `const` in these bits? I think the difference in functionality are but `const` is more idiomatic

Reply:
Agreed we’ll fix this later for the sake of not having to re-request reviews 😄